broadinstitute/pearl
# JN-275 adding viz of portal env. config

This adds some rough-and-ready display of more portal configs to help us troubleshoot and debug publishing/copying.  None of it is editable yet.

Added new links to site content,  and 'settings'
![image](https://user-images.githubusercontent.com/2800795/231510771-0df92c64-3879-4858-8024-80647d3f2006.png)

The 'Settings' page is the PortalEnvironmentConfig
![image](https://user-images.githubusercontent.com/2800795/231511111-4eb43824-69c9-4bb6-b4cf-7c0790b74dfa.png)

The site content takes you to a rough view of the html page configs.
![image](https://user-images.githubusercontent.com/2800795/231514777-b3994405-822a-4db8-845e-b2b3ca827284.png)

TO TEST:
1. go to https://localhost:3000/ourhealth/env/sandbox/siteContent, confirm you see the site configs
2. go to https://localhost:3000/ourhealth/env/sandbox/config, confirm you see the environment settings
